react/react2/week1/homework/src/App.js

```diff
@@ -23,28 +26,31 @@ function App() {
   useEffect(() => {
     fetchAPI()
   }, [fetchAPI])
+  const title = "Github user searcher"
+  const AppValues = [title]
   return (
     <div className="App">
-      <h1> Github user searcher </h1>
+      {/* All state should be managed using the context api! */}
+      <AppContext.Provider value={AppValues}>
+      <H1Title /> 
       {/* When the search is loading, show "loading..." */}
-      {loading ? "Loading" : 
+      {loading ? "loading..." : 
         <>
-          <input value={user} onChange={inputHandler} placeholder="Search for user" />
+          <input value={user} onChange={inputHandler} placeholder="Search for user" /><br />
```

Comment:
input should be visible at all times, currently, it's visible only when loading is false.